great_expectations_cloud/agent/actions/run_column_descriptive_metrics_action.py

```diff
@@ -28,6 +28,7 @@
     from great_expectations.experimental.metric_repository.metrics import MetricRun
 
 
+# TODO Refactor
```

Comment:
Needs to be DRYer

Reply:
I think we are ok in this case because both the `ColumnDescriptiveMetricsAction` and `MetricListAction` inherit from `AgentAction`, and we will eventually just have `MetricListAction` (and CDM would go away).  Happy to talk more or include in a follow-up PR.